dev/.rat-excludes

```diff
@@ -48,32 +44,17 @@ jquery.mustache.js
 pyspark-coverage-site/*
 cloudpickle/*
 join.py
-SparkExprTyper.scala
 SparkILoop.scala
-SparkILoopInit.scala
-SparkIMain.scala
-SparkImports.scala
-SparkJLineCompletion.scala
-SparkJLineReader.scala
-SparkMemberHandlers.scala
-SparkReplReporter.scala
 sbt
 sbt-launch-lib.bash
 plugins.sbt
-work
```

Comment:
This is for the Standalone worker daemon processes when we start a dev environment, we shall not clean it.